sql/patch_104_105_c.sql

```diff
@@ -0,0 +1,21 @@
+-- Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+-- Copyright [2016-2022] EMBL-European Bioinformatics Institute
+-- 
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+-- 
+--      http://www.apache.org/licenses/LICENSE-2.0
+-- 
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+ALTER TABLE genomic_feature_disease_comment ADD COLUMN is_public tinyint(1) NOT NULL DEFAULT '0';
```

Comment:
For consistency, you should add unsigned to the query (same as schema):
`is_public tinyint(1) unsigned NOT NULL DEFAULT '0';`

The same for the next command.